bundle/manifests/volsync.clusterserviceversion.yaml

```diff
@@ -203,6 +203,27 @@ spec:
           - patch
           - update
           - watch
+        - apiGroups:
+          - ""
+          resources:
+          - persistentvolumeclaims/finalizers
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - persistentvolumes
+          verbs:
+          - get
+          - list
+          - patch
+          - watch
```

Comment:
This one seems to be an OpenShift specific thing - when adding an ownerRef it gives you an error like:
`cannot set blockOwnerDeletion if an ownerReference refers to a resource you can't set finalizers on`.  Since we add an ownerRef to pvc' on the original pvc I would hit this error in OpenShift in the e2e tests.

May be related to the OwnerReferencesPermissionEnforcement plugin:  https://sdk.operatorframework.io/docs/faqs/#after-deploying-my-operator-why-do-i-see-errors-like-is-forbidden-cannot-set-blockownerdeletion-if-an-ownerreference-refers-to-a-resource-you-cant-set-finalizers-on-